exasol_sphinx_github_pages_generator/parser.py

```diff
@@ -1,10 +1,18 @@
+import sys
 import argparse
+from functools import partial
+
+
+class Console:
```

Comment:
Please put this class into its own module  to avoid coupling

Reply:
Hi @umitbuyuksahin,

I agree that a class with the name `Console` does not make to much sense in a `parser` module. I Intended to rename the module to `cli` so it would make more sense to have `Parser` and `Console` within there. I hesitate though because I saw you where also working on the `parser.py` module. But you're right so I think I rather rename now.